### PR TITLE
Add docs for Google Cloud BigQuery sink connector

### PR DESCRIPTION
**Pull request by Huanli-Meng**
### Modification

Add docs for the Google Cloud BigQuery sink connector:

- Affected releases: 2.8.3, 2.9.2, and 2.10.1
- Add a YAML file to sync docs between Google cloud BigQuery repo to SN Hub
- Add related images
**Note**
The content has been reviewed through the google doc.

### Preview link:
https://staging--streamnative-hub.netlify.app/connectors/google-bigquery-sink/2.9.2

### CODE REVIEW

**Review by shibd — APPROVED** [review]

File: connectors/google-bigquery-sink/2.10.1/google-bigquery-sink.md [hunk 1]
```diff
@@ -0,0 +1,408 @@
+---
+description: The Google Cloud BigQuery sink connector pulls data from Pulsar topics and persists data to to Google Cloud BigQuery tables.
```
Comment (teachwrite) [inline connectors/google-bigquery-sink/2.10.1/google-bigquery-sink.md:2]:
```suggestion
description: The Google Cloud BigQuery sink connector pulls data from Pulsar topics and persists data to Google Cloud BigQuery tables.
```

File: connectors/google-bigquery-sink/2.8.3/google-bigquery-sink.md [hunk 2]
```diff
@@ -0,0 +1,408 @@
+---
+description: The Google Cloud BigQuery sink connector pulls data from Pulsar topics and persists data to to Google Cloud BigQuery tables.
```
Comment (teachwrite) [inline connectors/google-bigquery-sink/2.8.3/google-bigquery-sink.md:2]:
```suggestion
description: The Google Cloud BigQuery sink connector pulls data from Pulsar topics and persists data to Google Cloud BigQuery tables.
```

File: connectors/google-bigquery-sink/2.9.2/google-bigquery-sink.md [hunk 3]
```diff
@@ -0,0 +1,408 @@
+---
+description: The Google Cloud BigQuery sink connector pulls data from Pulsar topics and persists data to to Google Cloud BigQuery tables.
```
Comment (teachwrite) [inline connectors/google-bigquery-sink/2.9.2/google-bigquery-sink.md:2]:
```suggestion
description: The Google Cloud BigQuery sink connector pulls data from Pulsar topics and persists data to Google Cloud BigQuery tables.
```

**Review by teachwrite — APPROVED** [review]
Looks good; just needs a couple of small typos fixed. Otherwise, this is approved for merging.

**Pull request merged**